orchestrator/careplanservice/service.go

```diff
@@ -87,6 +102,15 @@ func (s Service) readRequest(httpRequest *http.Request, target interface{}) erro
 	return json.Unmarshal(data, target)
 }
 
+func (s Service) notifySubscribers(ctx context.Context, resource interface{}) {
+	// Send notification for changed resources
+	notifyCtx, cancel := context.WithTimeout(ctx, subscriberNotificationTimeout)
```

Comment:
What requirements do we have for failure resolution/retry? If a notification fails to deliver, do we need a DLQ/Redrive or is it fine for the message to not be delivered?

Maybe we could implement a recursive retry with incremental backoff to start with if needed?

Reply:
Retry is out of scope for this PR, but the general thought is;
- it won't be retried by the CPS
- CPS could perform a heartbeat (it's spec'd in FHIR) to see if the subscriber is available
- CPC will periodically check at the CPS if its subscriber(s) are up-to-date

Reply:
Fine with me, I just wanted to make sure we had it in mind. Thanks!